### ISSUE TITLE
Error return when run responder.py

### ISSUE THREAD

**Issue by secfung** [lgandx/Responder#230]
Hi All,

I got this error when running responder.py on kail linux. Any clue for this issue? thank you!

#responder -I eth0 -A

Traceback (most recent call last):
  File "/usr/lib/python3.10/configparser.py", line 790, in get
    value = d[option]
  File "/usr/lib/python3.10/collections/__init__.py", line 986, in __getitem__
    return self.__missing__(key)            # support subclasses that define __missing__
  File "/usr/lib/python3.10/collections/__init__.py", line 978, in __missing__
    raise KeyError(key)
KeyError: 'dcerpc'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/share/responder/./Responder.py", line 58, in <module>
    settings.Config.populate(options)
  File "/usr/share/responder/settings.py", line 99, in populate
    self.DCERPC_On_Off      = self.toBool(config.get('Responder Core', 'DCERPC'))
  File "/usr/lib/python3.10/configparser.py", line 793, in get
    raise NoOptionError(option, section)
configparser.NoOptionError: No option 'dcerpc' in section: 'Responder Core'

**Comment by also-here** [comment]
Hi secfung,

I'm pretty sure the issues is/was that you are using an older Responder.conf file (referenced as 'Responder Core' in your error message). Essentially, lgandx has kindly added some fun new servers, e.g. DCERPC, WINRM, SNMP, and Responder.py is checking Responder.conf to see if they should be on or off. When it doesn't find them listed you get that error message. You can either grab a fresh copy of Responder.conf from here or, if you've hacked up your local copy and want to keep it, you can just add the missing options to the list of servers. If you are missing DCERPC you're likely missing others so make sure they're all there or we'll be right back here dealing with a slightly updated error message. 

TL;DR It's cuz of the Responder.conf file. Pull a new copy.

**Comment by secfung** [comment]
Thanks and the problem is fixed.
